getting-started/plugin-features/README.md

```diff
@@ -117,4 +118,25 @@ Spell checking is enabled for all identifiers, comments and markdown text. You c
 
 ![alt text](images/spell.gif)
 
+## Code folding
 
+Following code content can be folded.
+
+- Import declarations
+
+  ![alt text](images/importFolding.png)
+
+- Markdown documentation
+
+  ![alt text](images/markdownFolding.png)
+
+- Comments
+
+  ![alt text](images/commentFolding.png)
+- Braced content
+
+  ![alt text](images/braceFolding.png)
+
+- Brace pipe content
```

Comment:
IMO we can consider this as as braced content. Therefore no need to add a separate entry.